### DISCUSSION TITLE
CORE_RL_XX vs CORE_DB_xx

### DISCUSSION THREAD

**Discussion by jmls** [ImageMagick/ImageMagick6#254] [General] [answered]
I'm no c++ guy , but I have been tasked with converting one of our c++ projects from Visual Studio 2012 to 2022

One of the issues is that the project has a set of CORE_RL_xxx.lib files that are built using 2012. When I try and recompile imageMagic using visual studio , I get files with CORE_DB_xxx.lib

This may be a c++ question, a visual C question or an Imagemagic question .. 

What is the difference between the 2 files ? How can I generate CORE_RL_xxx files ? 

I'm hoping that it's just a setting or they are interchangeable ;) 

thanks

**Comment by dlemstra** [comment, marked as answer]
You will get a `CORE_DB` file with a debug build and a `CORE_RL` file with a release build.

**Comment by jmls** [comment]
ahhh fantastic , thank you so much
